src/components/organisms/OrganismArticle.vue

```diff
@@ -12,6 +12,7 @@
         v-for="article in articles"
         :key="article.$id"
         :article="article"
+        displayIcons="true"
```

Comment:
As this is a Boolean remove unnecessary definitions
```suggestion
        displayIcons
```